app/operations/story_operations.rb

```diff
@@ -0,0 +1,134 @@
+module StoryOperations
+
+  module MemberNotification
```

Comment:
I would be more in favor of moving these modules out in their own files. It really looks like a lot of things going on IMHO